### DISCUSSION TITLE
How to turn on active learning - epistemic uncertainty

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#893] [Q&A]
Hi there,

I'm using the latest stable versions of monai label (both v0.4.2 installed by pip and v0.4.0 by docker image), and try to turn on `epistemic uncertainty` in DeepEdit for radiology. I modified the config file of DeepEdit, change to `true` from `false`:

![Screenshot from 2022-07-25 16-32-32](https://user-images.githubusercontent.com/90806807/180891980-990f4aaa-6bd5-4046-842e-14cf5ba59c8e.png)

After that, when I start the server, the output shows it is turned on
``` 
[2022-07-25 16:23:42,642] [20251] [MainThread] [INFO] (lib.configs.deepedit:144) - EPISTEMIC Enabled: 1; Samples: 5
```
 but no calculation of the epistemic score, and in 3DSlicer, there's no `EPISTEMIC` option in the `Active Learning` section.

After I click `Next Sample` in 3DSlicer, I see this in server output:
```
[2022-07-25 23:45:07,511] [381] [MainThread] [INFO] (monailabel.endpoints.activelearning:42) - Active Learning Request: {'strategy': 'random', 'client_id': 'user-xyz'}
```
How to turn on active learning?

**Comment by diazandr3s** [comment]
Hi @hw-ju,

This is a good question.
Please add this line when starting the server:

> --conf skip_scoring false --conf skip_strategies false --conf epistemic_enabled true

Otherwise, change those flags within the main file: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/main.py#L153

**Reply by hw-ju** [reply]
@diazandr3s That works! Thanks!
Just to confirm, after using that line, we also need to choose `epistemic` in MONAI Label 3DSlicer plugin as in screenshot below, correct? I tried choosing `random` option in the plugin, and it began fetching randomly.
![image](https://user-images.githubusercontent.com/90806807/193705452-aceb7c45-1043-43a2-aed3-53ad4538bdeb.png)

**Reply by diazandr3s** [reply]
That's correct! Thanks for the update :)
